examples/tv-casting-app/tv-casting-common/support/CastingStore.h

```diff
@@ -83,6 +83,7 @@ class CastingStore : public chip::FabricTable::Delegate
         kCastingPlayerPortTag,
         kCastingPlayerInstanceNameTag,
         kCastingPlayerDeviceNameTag,
+        kCastingPlayerCommissionerPasscodeTag,
```

Comment:
This re-orders/re-names all the other tags. Is this expected ? I would expect TLV tags to be fixed as long as there is a possiblity of old data to be stored or received (e.g. by combining new and old client/server applications).

Is this not a concern here? If so there should be a comment associated with this enum otherwise you have backwards compatibility issues.

Reply:
I'll move it to the end in the follow up commit.